target/linux/mediatek/dts/mt7981b-iptime-ax3000m.dts

```diff
@@ -0,0 +1,270 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+
+/dts-v1/;
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include "mt7981.dtsi"
+
+/ {
+	model = "ipTIME AX3000M";
+	compatible = "iptime,ax3000m", "mediatek,mt7981-spim-snand-rfb";
+
+	aliases {
+		label-mac-device = &gmac0;
+		led-boot = &led_cpu;
+		led-failsafe = &led_cpu;
+		led-running = &led_cpu;
+		led-upgrade = &led_cpu;
+		serial0 = &uart0;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+		bootargs-append = " ubi.mtd=ubi rootfstype=squashfs";
+		bootargs-replace = "boot_from=B|ubi.block=0,rootfs2 root=/dev/ubiblock0_4";
+	};
+
+	memory {
+		reg = <0x0 0x40000000 0x0 0x10000000>;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_cpu: cpu {
+			function = LED_FUNCTION_CPU;
+			color = <LED_COLOR_ID_AMBER>;
+			gpios = <&pio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan2g {
+			label = "amber:wlan2g";
+			gpios = <&pio 34 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		wlan5g {
+			label = "amber:wlan5g";
+			gpios = <&pio 35 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
+		};
```

Comment:
Please use `color`/`function` properties instead of `label`.

- `LED_FUNCTION_WLAN_2GHZ`
- `LED_FUNCTION_WLAN_5GHZ`

Reply:
fixed.